word_generation/generator.py

```diff
@@ -15,8 +15,9 @@
 initiated = False
 MAX_LENGTH = 7
 
+
 def init():
-    with open ('trigram_counts_full.csv') as file:
+    with open('word_generation/trigram_counts_full.csv') as file:
```

Comment:
So I found out that relative paths are relative to the directory that the file is run from. So running it from this directory could read straight to the trigram file, but when app.py is run from the parent directory, the directory needs to be added.

Reply:
Thanks for fixing that, I always forget about the relative path thing.